frontend/client/src/components/Modal.js

```diff
@@ -19,6 +24,7 @@ const Modal = (props) => {
             <img src={close} alt="Close" />
           </span>
         )}
+        <h3>{props.title}</h3>
         {props.children}
```

Comment:
I think we can remove this, doesn't seem to be used anywhere else

Reply:
I was wrong, it turns out `props.children` is a special value, see [here](https://reactgo.com/react-props-children/)